evidence/alin/0005-alin-promote-II-2025.md

```diff
@@ -0,0 +1,177 @@
+which was a critical bottleneck. This was a very complex endeavour that I owned.
+The resulting updates are captured in 
+[PR 4035](https://github.com/paritytech/polkadot-sdk/pull/4035) and 
+[PR 4937](https://github.com/paritytech/polkadot-sdk/pull/4937).
+
+Elastic scaling also required updates to Cumulus, including support in the PoV recovery component, 
+which I delivered in [PR 4733](https://github.com/paritytech/polkadot-sdk/pull/4733). To support 
+adoption, I authored a [comprehensive documentation 
+guide](https://github.com/paritytech/polkadot-sdk/pull/4663) for parachain teams.
+
+Getting the elastic scaling feature from the MVP to the production status involved two threads of work: RFC103 and
+deprecating AsyncBackingParams.
+
+[RFC103](https://github.com/polkadot-fellows/RFCs/pull/103) addresses a potential vulnerability 
+where a collator could distribute the same collation to all backing groups. I contributed
+numerous times to the node-side implementation (https://github.com/paritytech/polkadot-sdk/pull/6011, https://github.com/paritytech/polkadot-sdk/pull/5908)
+and owned the Cumulus-side support: https://github.com/paritytech/polkadot-sdk/pull/5372, https://github.com/paritytech/polkadot-sdk/pull/5888.
+
+To enable dynamic resource allocation, I [removed support for scheduling 
+TTL](https://github.com/paritytech/polkadot-sdk/pull/5461) from the coretime assignment pallet and 
+[deprecated `AsyncBackingParams`](https://github.com/paritytech/polkadot-sdk/pull/7254), which had 
+become obsolete with the introduction of the claim queue concept. Removing these static parameters 
+reduced validator overhead and improved spam resistance.
+
+In parallel, I participated in the design of [streamlined block 
+production](https://github.com/paritytech/polkadot-sdk/issues/5190#issuecomment-2505949587), proposing 
+a simplified post-state-based approach. I developed a [proof of 
+concept](https://github.com/paritytech/polkadot-sdk/tree/alindima/post-state-poc) and tested it using 
+gluttons and spammening parachains. While the need for this implementation hasn’t yet materialized, 
+the groundwork remains valuable for future scaling needs and showcases my ability to innovate and meaningfully
+contribute to areas outside of my previous expertise.
+
+### Collator protocol revamp
+
+In an effort to enhance the reliability of the parachain backing process, I started leading a 
+comprehensive revamp of the collator protocol, introducing a persistent reputation system for collators.
+This project builds on previous issues raised in the 
+ecosystem ([reference](https://github.com/paritytech/polkadot-sdk/issues/616)) and collaborative 
+discussions with other fellows.
+
+I created a detailed proposal, currently under private review (which will soon make it to an RFC),
+and outlined the development process (which I'm managing) in 
+an open [project plan](https://github.com/orgs/paritytech/projects/119/views/2).
```

Comment:
More of a general comment rather than feedback on the promotion letter:

It seems weird that there is already a public project plan and some implementation work already done, _before_ the proposal/design was discussed and approved in public by the Fellowship and any interested community members.